### PR TITLE
style(GlobalHeader): Use theme variable for header height

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

Time for some magic.

* Uses headerHeight variable.

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604192db970aba3df89f519d--gamut-preview.netlify.app>https://604192db970aba3df89f519d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604192db970aba3df89f519d>Deploy Logs</a></p>

File: packages/gamut-labs/src/experimental/GlobalHeader/index.tsx [hunk 1]
```diff
@@ -93,22 +94,28 @@ const getMobileAppHeaderItems = (
...
-    </Box>
-  </>
-);
+export const GlobalHeader: React.FC<GlobalHeaderProps> = (props) => {
+  const theme = useTheme();
+  return (
+    <>
+      <Box
+        display={{ base: 'none', md: 'block' }}
+        height={theme.elements.headerHeight}
```
Comment (saghdaey) [inline packages/gamut-labs/src/experimental/GlobalHeader/index.tsx:103]: wowowow!! wait did @katiezutter know you were working on this

**Review by saghdaey — APPROVED** [review]
b-e-a-utiful

File: packages/gamut-labs/src/experimental/GlobalHeader/index.tsx (hunk 1 above)
Comment (katiezutter) [reply packages/gamut-labs/src/experimental/GlobalHeader/index.tsx:103]: 😱

**Review by katiezutter — APPROVED** [review]
This seriously made my day 🙌

**Review by christian-dinh — APPROVED** [review]
hell yea 🔥 

I guess I can just close https://github.com/Codecademy/client-modules/pull/1417 then?

**Comment by codecaaron** [comment]
> hell yea 🔥
> 
> I guess I can just close #1417 then?

If you still are gonna use the utility you can just pull this one into that and we can get these headers ANIMATED :)

**Review by dreamwasp — APPROVED** [review]
🔥

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.2.2-alpha.bf1678.0<br/>@codecademy/gamut-labs@10.5.1-alpha.bf1678.0<br/>@codecademy/styleguide@30.1.1-alpha.bf1678.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604288f986e17323119ba350--gamut-preview.netlify.app>https://604288f986e17323119ba350--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604288f986e17323119ba350>Deploy Logs</a></p>

**Pull request merged**